queries/lua/folds.scm

```diff
@@ -1,12 +1,1496 @@
-[
- (for_in_statement)
- (for_statement)
- (while_statement)
- (repeat_statement)
- (if_statement)
- (do_statement)
- (function_definition)
- (local_function)
- (function)
- (table)
-] @fold
+
+
+
+
+
+
+<!DOCTYPE html>
```

Comment:
@clason probably you wanted to download the raw document. Our users won't be to happy with HTML

Reply:
🤦🏻